src/Accelerators/NNPA/Conversion/ONNXToZHigh/NNPALimit.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//===----------------------- NNPALimit.cpp --------------------------------===//
+//
+// Copyright 2022 The IBM Research Authors.
+//
+// =============================================================================
+//
+// The NNPA maximum supported dimension index size value by using
+// zdnn_get_nnpa_max_dim_idx_size() This value depends on HW.
+//
+//===----------------------------------------------------------------------===//
+
+// The NNPA maximum supported dimension index size value by using
+// zdnn_get_nnpa_max_dim_idx_size() This value depends on HW.
+#define DLCPP_MAXIMUM_DIMENSION_INDEX_SIZE 32768
```

Comment:
Instead of using C macos to define a constant I suggest we use a proper `static constexpr` variables. Macros are hard to debug.